From 4f3436117435d853057b608fa0ba5c1c204956be Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Uwe=20Kleine-K=C3=B6nig=20=28The=20Capable=20Hub=29?=
 <u.kleine-koenig@baylibre.com>
Date: Fri, 3 Jul 2026 17:46:26 +0200
Subject: [PATCH 13282/15333] nfc: Unify style of usb_device_id arrays
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The usual coding style is to skip the comma after a initializer iff the
closing } is on the same line. Also there is usually no empty line
between the array and the MODULE_DEVICE_TABLE() macro.

Adapt two drivers accordingly to match this common style.

Signed-off-by: Uwe Kleine-König (The Capable Hub) <u.kleine-koenig@baylibre.com>
Link: https://patch.msgid.link/8a186cb0376deb3d4f4264e6ed351562b79bb53d.1783091699.git.u.kleine-koenig@baylibre.com
Signed-off-by: David Heidelberg <david@ixit.cz>
---
 drivers/nfc/nfcmrvl/usb.c | 1 -
 drivers/nfc/port100.c     | 4 ++--
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/drivers/nfc/nfcmrvl/usb.c b/drivers/nfc/nfcmrvl/usb.c
index 4babde8e4249..c7f2afe00b93 100644
--- a/drivers/nfc/nfcmrvl/usb.c
+++ b/drivers/nfc/nfcmrvl/usb.c
@@ -17,7 +17,6 @@ static struct usb_device_id nfcmrvl_table[] = {
 					USB_CLASS_VENDOR_SPEC, 4, 1) },
 	{ }	/* Terminating entry */
 };
-
 MODULE_DEVICE_TABLE(usb, nfcmrvl_table);
 
 #define NFCMRVL_USB_BULK_RUNNING	1
diff --git a/drivers/nfc/port100.c b/drivers/nfc/port100.c
index 5ae61d7ebcfe..b613f5e2fd57 100644
--- a/drivers/nfc/port100.c
+++ b/drivers/nfc/port100.c
@@ -1480,8 +1480,8 @@ static const struct nfc_digital_ops port100_digital_ops = {
 };
 
 static const struct usb_device_id port100_table[] = {
-	{ USB_DEVICE(SONY_VENDOR_ID, RCS380S_PRODUCT_ID), },
-	{ USB_DEVICE(SONY_VENDOR_ID, RCS380P_PRODUCT_ID), },
+	{ USB_DEVICE(SONY_VENDOR_ID, RCS380S_PRODUCT_ID) },
+	{ USB_DEVICE(SONY_VENDOR_ID, RCS380P_PRODUCT_ID) },
 	{ }
 };
 MODULE_DEVICE_TABLE(usb, port100_table);
-- 
2.55.0

